src/cmd/cmd_wasm2wat.ml

```diff
@@ -4,13 +4,19 @@
 
 open Syntax
 
-let cmd_one file =
+(* Utility function to handle writing to a file or printing to stdout *)
```

Comment:
If you want this to be an item comment for the function, you should remove the empty line between the comment and the `let cmd_one ...` line.

Also, you must replace `(*` by `(**`. see the [manual](https://ocaml.org/manual/5.2/doccomments.html#ss:item-comments).